modules/supported-platforms-for-managed-clusters.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: PROCEDURE
+[id="supported-platforms-for-managed-clusters_{context}"]
+= supported-platforms-for-managed-clusters
+
+You can manually specify the platform type for managed clusters if you do not want the platform type to be set automatically.
+
+.Prerequisites
+
+The Assisted Installer currently supports the following OpenShift platforms:
+- BareMetal
+- VSphere
+- None
+
+.Procedure
+
+. Select the platform in the AgentClusterInstall CR, via `spec.platformType`.
+
+In some cases, the system will automatically set the platform type:
+- The default platform for multi-node clusters is BareMetal
+- The default platform for Single Node OpenShift is None
+- Enabling `spec.networking.userManagedNetworking` without specifying the platform will cause the platform to be `None`.
+
+BareMetal platform and `userManagedNetworking` are not compatible.
+
+[NOTE]
+====
+vSphere is not supported for single node OpenShift and uses UPI installation.
+Virtual Machine (VM) provisioning should be done by the user.
+VIPs DHCP allocation should be configured manually in vSphere.
+====
+
+.Verification
+
+Verify the selected platform in the AgentClusterInstall CR, via `status.platformType`.
```

Comment:
@jelkosz and @trewest, do you have an example code snippet that I can use when verifying the selected platform?

Reply:
```
$ oc get AgentClusterInstall cluster -n cluster -o jsonpath='{.status.platformType}{"\n"}'
VSphere
```

Reply:
Updated with requested code snippet.